tests/installation/boot_into_snapshot.pm

```diff
@@ -38,6 +38,11 @@ sub run() {
             die "OS_VERSION after Rollback matches OS_VERSION before Rollback";
         }
     }
+    if (get_var('CHECK_SNAPPER_ROLLBACK_CLEANUP')) {
+        assert_script_run('snapper rollback');
+        assert_script_run('snapper --iso list');
+        assert_screen('snapper-autocleanup');
```

Comment:
can you remind me how it does look like before and after the feature being done?

Is it

```perl
assert_script_run('snapper --iso list | grep \'number.*rollback\'.*important=yes',
    fail_message => 'rollback backup should be marked for cleanup');
```

then?